sites/default/menu_data.json

```diff
@@ -551,7 +559,26 @@
             "url" : "/interface/reports/patient_list.php",
             "children" : [ ],
             "requirement" : 0
+
         }, {
+            "label" : "Clinical Statistics: Demographics vs. Diagnosis",
+            "menu_id" : "clndd0",
+            "target" : "clndd",
+            "url" : "/interface/reports/clinical_stats_by_demographics_report.php",
+            "children" : [ ],
+            "requirement" : 0
+        },
+            {
+            "label" : "Clinical Statistics: Demographics vs. Lab Results",
+            "menu_id" : "clndl0",
```

Comment:
inconsistent, the above is dd0 and this one is d10  ?  shouldn't be dd0 and dd1 ?

Reply:
Done